ui: show coordinator node ID on job detail

dt opened the pull request:
Previously you could find this from the table view only and had to toggle it on to do so, but it is often useful if you are inspcting a specific job to know where it is running.

Release note: none.
Epic: none.

dt committed:
ui: show coordinator node ID on job detail

Previously you could find this from the table view only and had to toggle it on to do so,
but it is often useful if you are inspcting a specific job to know where it is running.

Release note: none.
Epic: none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/141986)
<!-- Reviewable:end -->

dt commented:
<img width="632" alt="Screenshot 2025-02-25 at 09 28 18" src="https://github.com/user-attachments/assets/53251a81-8940-4f4e-8d0f-9a5f7c4c66cd" />

dt committed:
ui: show job detail times in seconds

Jobs run for minutes, hours or even days, and take multiple seconds at least to start, update
or stop, so millisecond precision on their timestamps is just noise rather than being
useful information for the time scale on which jobs operate.

Release note: none.
Epic: none.

kev-cao reviewed (approved):
LGTM!

dt commented:
TFTR!

bors r+
